app/javascript/components/widgets/weather/semi-circle.jsx

```diff
@@ -7,6 +7,28 @@ import {
   timeAndDateForTimezone,
 } from '../../../lib/datetime';
 import { colors } from '../../../lib/theme';
+import MoonPhase from './moon-phase';
+
+const getPhaseType = phase => {
+  switch (true) {
+    case phase === 0:
```

Comment:
`Java` has the restriction that the `switch` condition must be an `int` (and now `int | String`) but JS will take whatever kind of value you throw at it (tho it won't do deep equality checks for `Object` values).

Reply:
Yeah, stick with if/else if/else here as well.